### PR TITLE
Run ava tests in a gulp task

### PR DESCRIPTION
**Pull request by rsimha-amp**
#10482

### CODE REVIEW

File: build-system/tasks/ava.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+/**
+ * Runs ava tests.
+ */
+function runAvaTests() {
+	return gulp.src([
+    'csvify-size/test.js',
+    'get-zindex/test.js',
+    'prepend-global/test.js'
+  ])
+  .pipe(ava({ silent: process.env.TRAVIS ? true : false }));
```
Comment (choumx) [inline build-system/tasks/ava.js]: Nit: `{silent: !!process.env.TRAVIS}`

Comment (rsimha-amp) [inline build-system/tasks/ava.js]: Done.

**Comment by rsimha-amp** [comment]
Earlier, we were running `npm run ava` using `npm` version 2.x, which does not support the `--silent` flag. Moving to `gulp-ava` serves two purposes:

1. We can silence logs as I've done here.
2. It is now possible for other larger sets of tests to include these tests by expressing a dependency on the `ava` task.

File: build-system/tasks/ava.js [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+'use strict';
+
+var gulp = require('gulp-help')(require('gulp'));
+const ava = require('gulp-ava');
+
+/**
+ * Runs ava tests.
+ */
+function runAvaTests() {
+	return gulp.src([
```
Comment (choumx) [inline build-system/tasks/ava.js]: -2 spaces

File: build-system/tasks/ava.js [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+'use strict';
+
+var gulp = require('gulp-help')(require('gulp'));
```
Comment (choumx) [inline build-system/tasks/ava.js]: `const`

Comment (rsimha-amp) [inline build-system/tasks/ava.js]: Fixed.

File: build-system/tasks/ava.js (hunk 2 above)
Comment (rsimha-amp) [inline build-system/tasks/ava.js]: It was a stray tab. Fixed.

**Pull request merged**